.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2026-03-14 - explicit focus visible rules
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)